Documentation/helm-values.rst

```diff
@@ -17,6 +17,10 @@
      - Install the cilium agent resources.
      - bool
      - ``true``
+   * - agentNotReadyTaintKey
+     - Configure the key of the taint indicating that Cilium is not ready on the node. When set to a value starting with ``ignore-taint.cluster-autoscaler.kubernetes.io/``\ , the Cluster Autoscaler will ignore the taint on it's decisions, allowing the cluster to scale up. when simulating scheduling, allowing you to scale up from zero.
```

Comment:
```suggestion
     - Configure the key of the taint indicating that Cilium is not ready on the node. When set to a value starting with ``ignore-taint.cluster-autoscaler.kubernetes.io/``\ , the Cluster Autoscaler will ignore the taint on it's decisions, allowing the cluster to scale up.
```

Reply:
In theory this change should be made automatically by `make -C Documentation helm-docs`, but since you can't run that I'm suggesting what I believe would be the end result.